docs-client/src/containers/MethodPage/DebugPage.tsx

```diff
@@ -504,9 +508,11 @@ class DebugPage extends React.PureComponent<Props, State> {
 
     let urlRequestBody;
     if (this.props.useRequestBody) {
-      urlRequestBody = urlParams.has('request_body')
-        ? jsonPrettify(urlParams.get('request_body')!)
-        : undefined;
+      urlRequestBody = undefined;
```

Comment:
I guess we can remove this statement. 😀

Reply:
I'll remove this statement. Thank you for review!